File: src/main/resources/template/vr/VrLauncher.java
package ${package};

import com.jme3.app.VRAppState;
import com.jme3.app.VREnvironment;
import com.jme3.system.AppSettings;

class VrLauncher extends VRAppState  {
    public VrLauncher(AppSettings settings, VREnvironment environment) {
        super(settings, environment);
        Main app=new Main();
        initialize(app.getStateManager(),app);
        getApplication().start();
    }
}

File: src/main/resources/template/vr/VrLauncher.kt
package ${package}

import com.jme3.app.VRAppState
import com.jme3.app.VREnvironment
import com.jme3.system.AppSettings

internal class VrLauncher(settings: AppSettings, environment: VREnvironment) : VRAppState(settings, environment) {
    init {
        val app = Main()
        initialize(app.getStateManager(), app)
        getApplication().start()
    }
}